### ISSUE TITLE
App Crashes Randomly after published

### ISSUE THREAD

**Issue by kamalpandey** [expo/expo#9696]
### Summary of Issue 
App crash randomly during startup

### Environment - output of `expo diagnostics` & the platform(s) you're targeting
SDK 38

Below is the log 

[logcat1.txt](https://github.com/expo/expo/files/5061262/logcat1.txt)

```
dependencies": {
    "@react-native-community/masked-view": "^0.1.5",
    "@react-native-community/netinfo": "^5.9.2",
    "@react-native-community/viewpager": "^3.3.0",
    "axios": "^0.19.2",
    "decode-html": "^2.0.0",
    "expo": "^38.0.0",
    "expo-analytics-segment": "~8.2.1",
    "expo-apple-authentication": "~2.2.1",
    "expo-asset": "~8.1.7",
    "expo-calendar": "~8.2.1",
    "expo-constants": "~9.1.1",
    "expo-font": "~8.2.1",
    "expo-google-app-auth": "^7.0.0",
    "expo-google-sign-in": "~8.2.1",
    "expo-image-picker": "~8.3.0",
    "expo-linking": "^1.0.3",
    "expo-notifications": "~0.3.3",
    "expo-permissions": "~9.0.1",
    "expo-web-browser": "~8.3.1",
    "json-api-normalizer": "^1.0.0",
    "lodash": "^4.17.10",
    "moment": "^2.27.0",
    "pusher-js": "^6.0.3",
    "raygun4js": "2.16.1",
    "react": "16.11.0",
    "react-native": "https://github.com/expo/react-native/archive/sdk-38.0.2.tar.gz",
    "react-native-autocomplete-input": "^4.2.0",
    "react-native-collapsible": "^1.5.2",
    "react-native-confirmation-code-field": "^6.5.0",
    "react-native-country-picker-modal": "^2.0.0",
    "react-native-device-detection": "^0.2.0",
    "react-native-dropdownalert": "^4.3.0",
    "react-native-elements": "^1.1.0",
    "react-native-gesture-handler": "~1.6.0",
    "react-native-gifted-chat": "^0.16.3",
    "react-native-htmlview": "0.13.0",
    "react-native-image-progress": "^1.1.1",
    "react-native-intl-phone-input": "1.2.12",
    "react-native-keyboard-aware-scroll-view": "^0.9.1",
    "react-native-keyboard-spacer": "^0.4.1",
    "react-native-loading-spinner-overlay": "^1.0.1",
    "react-native-mentions": "1.1.4",
    "react-native-modal": "^11.5.6",
    "react-native-phone-input": "^0.2.4",
    "react-native-popup-menu": "^0.15.9",
    "react-native-progress": "^4.1.2",
    "react-native-read-more-text": "^1.1.2",
    "react-native-reanimated": "~1.9.0",
    "react-native-render-html": "4.2.0",
    "react-native-safe-area-context": "~3.0.7",
    "react-native-safe-area-view": "^1.1.1",
    "react-native-screens": "~2.9.0",
    "react-native-segmented-control-tab": "^3.4.1",
    "react-native-select-list2": "^1.0.8",
    "react-native-tab-view": "2.15.0",
    "react-native-tab-view-viewpager-adapter": "^1.1.0",
    "react-native-tag-input": "0.0.18",
    "react-native-webview": "9.4.0",
    "react-navigation": "^4.0.9",
    "react-navigation-stack": "^2.8.2",
    "react-navigation-tabs": "^2.9.0",
    "react-redux": "^7.0.0",
    "redux": "^4.0.5",
    "redux-form": "^8.3.6",
    "redux-object": "^0.5.10",
    "redux-persist": "^5.10.0",
    "redux-promise-middleware": "^6.1.1",
    "redux-thunk": "^2.3.0",
    "rn-placeholder": "^3.0.0",
    "sentry-expo": "~2.1.1",
    "styled-components": "^5.1.1",
    "uuid": "^3.2.1"
  },
  "devDependencies": {
    "axios-mock-adapter": "^1.16.0",
    "babel-eslint": "^10.0.2",
    "babel-preset-expo": "^8.2.3",
    "enzyme": "^3.10.0",
    "enzyme-adapter-react-16": "^1.14.0",
    "eslint": "^5.16.0",
    "eslint-config-airbnb": "^17.1.0",
    "eslint-config-prettier": "^5.0.0",
    "eslint-plugin-import": "^2.17.3",
    "eslint-plugin-jsx-a11y": "^6.2.1",
    "eslint-plugin-prettier": "^3.1.0",
    "eslint-plugin-react": "^7.13.0",
    "jest-expo": "^38.0.0",
    "jest-serializer-enzyme": "^1.0.0",
    "prettier": "^1.18.2",
    "react-addons-test-utils": "^15.6.2",
    "react-dom": "16.11.0",
    "reactotron-react-native": "^3.6.4",
    "redux-devtools-extension": "^2.13.8",
    "redux-logger": "^3.0.6",
    "redux-mock-store": "^1.5.3"
  }

```

**Comment by brentvatne** [comment]
https://docs.expo.io/workflow/debugging/#production-errors

if you managed to create a [mcve](https://stackoverflow.com/help/mcve) please feel free to share in a new issue.

**Comment by kamalpandey** [comment]
@brentvatne If I would have found any clue I would have resolved the issue.

**Comment by summerkiflain** [comment]
@kamalpandey did you find any solution? I am also facing same error:
```
signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x8
```
for me its happening in expo client and standalone version, no idea why its happening it was working fine before.

**Comment by Slapbox** [comment]
@brentvatne thanks for the tip, looks like I upgraded just before that came out.

I managed to reproduce this using `--nov-dev --minify`, but I don't think that gets us much further along at all. Any advice for trying to debug this? For now it's going to be slicing and dicing our app into smaller and smaller pieces? Like can we potentially use `--minify` without `--no-dev` and be able to debug in any meaningful fashion?
